libcodechecker/arg_handler.py

```diff
@@ -134,7 +134,7 @@ def handle_server(args):
                              str(instance['port'])))
 
         print("Your running CodeChecker servers:")
-        util.print_table(rows)
+        print(util.twodim_to_table(rows))
```

Comment:
Not possible. The twodim is well-formatted, with each row being a `2` or `3`-tuple. (Based on `instances_on_multiple_hosts`' value.)